api/CSS.json

```diff
@@ -0,0 +1,1534 @@
+{
+  "api": {
+    "CSS": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSS",
```

Comment:
This… Doesn’t sound right.

And doesn’t match [the existing table](https://developer.mozilla.org/en-US/docs/Web/API/CSS#Browser_compatibility) or #2167.

Reply:
Yeah, the spec mentioned above declares a partial `CSS` namespace with a few properties. The other part of that `CSS` interface is indeed in PR #2167, which was opened earlier, so I'm going to merge that one first and we'll need to rebase here (and take my comment about the missing properties into account).